scripts/zones/Windurst_Walls/npcs/Arbitrix.lua

```diff
@@ -0,0 +1,54 @@
+-----------------------------------
+-- Area: Windurst Woods
+--  NPC: Arbitrix
+-- Gobbie Mystery Box
+-- !pos -215.5 0.0 -147.3
+-----------------------------------
+local ID = require("scripts/zones/Windurst_Walls/IDs")
+require("scripts/globals/settings")
+-----------------------------------
+
+function onTrigger(player,npc)
+    local playerAgeDays = (os.time() - player:getTimeCreated()) / 86400
+
+    local dailyTallyPoints = player:getCurrency("daily_tally")
+    local firstVisit = dailyTallyPoints == -1
+
+    local gobbieBoxUsed = player:getCharVar("gobbieBoxUsed")
+    local specialDialUsed = 1
+    local adoulinDialUsed = 1
+    local pictlogicaDialUsed = 1
+    local wantedDialUsed = 1
```

Comment:
I know the dial variables are currently placeholders - so I'm noting for future development - but we might be able to combine and store all of these into a single bitmask.